components/mediaUploader.html

```diff
@@ -1,4 +1,5 @@
 <div class="media-uploader">
+    <script src="assets/1/0/bundles/sitetheorystratus/stratus/components/tag.js"></script>
```

Comment:
This will cause a secondary load of a Require.js module.  You need to put this in the `define` on your `mediaUploader.js` component.

Reply:
I'll take care of it in a followup commit.